src/main/java/application/rest/v1/configmaps/ConfigMapCache.java

```diff
@@ -191,8 +193,30 @@ public static V1ConfigMap getConfigMap(ApiClient client, String namespace, Strin
         try {
             CoreV1Api api = new CoreV1Api();
             api.setApiClient(client);
-
             V1ConfigMap map = api.readNamespacedConfigMap(name, namespace, null, null, null);
+
+            // issue warning if kappnav.io/map-type label is not set
+            V1ObjectMeta metadata = map.getMetadata();
+            if (metadata != null) {
+                Map labels = metadata.getLabels();
+                if (labels != null) {
+                    String mapType = (String) labels.get("kappnav.io/map-type");
+                    if (mapType == null) {
+                        if (Logger.isWarningEnabled()) {
+                            Logger.log(CLASS_NAME, "getConfigMap", Logger.LogType.WARNING, 
+                                "Label kappnav.io/map-type on ConfigMap Name: " + name + ", Namespace: " 
```

Comment:
Would suggest expanding the message to describe the impact of the missing label. e.g. "This ConfigMap is not being watched so it's cache entry may become stale and not contain the current resource from the cluster."

Reply:
code updated.